styles.css

```diff
@@ -3,6 +3,241 @@
   background-color: #191765;
 }
 
+.line {
+  padding: 24px;
+}
+
+.socialmedia {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 48px;
+  padding-right: 48px;
+  height: 40px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  list-style-type: none;
+}
+
+#language {
+  color: #9490ff;
+  letter-spacing: 0.03em;
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.language {
+  color: #f5f5ff;
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  gap: 4.78px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  list-style-type: none;
+  letter-spacing: 0.03em;
+}
+
+.languageframeworks {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.skill {
+  display: flex;
+  flex-direction: column;
+}
+
+.scroll {
+  display: flex;
+  justify-content: center;
+  gap: 12px;
+  padding-top: 32px;
+  padding-bottom: 32px;
+}
+
+.text {
+  width: 327px;
+  height: 104px;
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #9490ff;
+  padding-left: 12px;
+}
+
+.ellipse {
+  position: absolute;
+  padding-left: 290px;
+}
+
+.grid-container {
+  display: grid;
+  background-color: inherit;
+  padding: 10px;
+}
```

Comment:
- [ ] Kindly make sure to align each card's content using the CSS `flex-box` property as it's one of the project requirements.